brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Audit all the Sliders & make them responsive where there is needed.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/fa7cb4d8-c9d2-4c06-a50f-4d18960f54df?redirect=true&view=06c01065-22db-4b44-9a83-f0eceb06e66a&t=attributes
- Added slider in all blocks except deprecated once i.e Column, Section, Search.

### Screenshots


### Types of changes




- New feature (non-breaking change which adds functionality)

### How has this been tested?

- Drag and drop each block check slider is it responsive.
- If yes check on the responsive devices editor as well as on the front.
- Check to debug log, console, each block should be rendered properly.
- Setting should work properly.

### Checklist:
